src/Cafe/HW/Espresso/PPCTimer.cpp

```diff
@@ -7,9 +7,11 @@
 
 #if BOOST_OS_LINUX || BOOST_OS_MACOS
 static __inline__
-unsigned __int64 _umul128(unsigned __int64,
-                          unsigned __int64,
-                          unsigned __int64*);
+uint64 _umul128(uint64 multiplier, uint64 multiplicand, uint64 *highProduct) {
```

Comment:
Is `uint64` standard? Why not `uint64_t`?

Reply:
I think it's for consistency. (Since uint64 is used throughout the code)

Reply:
I would personally replace all instances of `uint64` with `uint64_t`, but it's not a big deal obviously :)

Reply:
> I think it's for consistency. (Since uint64 is used throughout the code)

Yep, that was my reasoning.